digital_asset_types/src/dapi/get_asset.rs

```diff
@@ -1,10 +1,33 @@
+use super::common::{asset_to_rpc, build_asset_response};
+use crate::{
+    dao::{scopes, Pagination},
+    rpc::{display_options::DisplayOptions, Asset},
+};
 use sea_orm::{DatabaseConnection, DbErr};
+use std::collections::HashMap;
 
-use crate::{dao::scopes, rpc::Asset};
-
-use super::common::asset_to_rpc;
-
-pub async fn get_asset(db: &DatabaseConnection, id: Vec<u8>) -> Result<Asset, DbErr> {
+pub async fn get_asset(
+    db: &DatabaseConnection,
+    id: Vec<u8>,
+    display_options: &DisplayOptions,
+) -> Result<Asset, DbErr> {
     let asset = scopes::asset::get_by_id(db, id, false).await?;
-    asset_to_rpc(asset)
+    asset_to_rpc(asset, display_options)
+}
+
+pub async fn get_asset_batch(
```

Comment:
We need to keep a name pattern for handlers, so the API does not look over-complicated. So far we have `getAssets*` methods that return multiple assets – ideally we should keep with this, so `get_asset_batch` would be names `get_assets`.

Reply:
We can add an alias if people feel strongly about this. I don't want to break anything for existing Helius users. We have quite a bit of traffic on these endpoints.

Reply:
I think we agreed to maintain the consistency with existing methods on the call. So we would have `get_assets` and `get_asset_proofs`. The alias seems a good solutions to not break existing Helius users.

Reply:
Addressed